docs/user-guide/admin-cli.md

```diff
@@ -401,7 +401,10 @@ optional arguments:
   --name NAME           Tenant name
   --volume-maxsize Num{MB,GB,TB} - e.g. 2TB
                         Maximum size of the volume that can be created
-  --allow-create        Allow create and delete on datastore if set to True
+  --allow-create ALLOW_CREATE
+                        Allow create and delete on datastore if set to True;
```

Comment:
As we discussed, we will address the change to make add and set commands consistent separately.